### PR TITLE
Update swagger with GitHub Action related changes.

### PR DESCRIPTION
**Pull request by nertim**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
It appears to be a breaking change on stable version.

As identified in the automated checks there are breaking changes, Please follow Azure Breaking changes approval request form as defined @ Breaking Change Process

File: specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json [hunk 1]
```diff
@@ -17041,75 +17046,6 @@
...
-          },
-          "default": {
-            "description": "App Service error response.",
-            "schema": {
-              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
-            }
-          }
-        }
-      }
     },
```
[the remark this answers is not in the document]
Comment (nertim) [reply specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json:17049]: synced offline.. will readd.

**Review by naveedaz — APPROVED** [review]

**Comment by nertim** [comment]
@weidongxu-microsoft - I am confused on what the breaking change is here? the added properties are optional, so they should not be considered as breaking changes.

**Comment by weidongxu-microsoft** [comment]
@nertim Adding update-able new property **IS** considered to be breaking changes.

Doc from ARM: https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html
See "Adding properties within an API version"

It is for service team to explain whether the implementation can handle it or not. Hence the question.

**Comment by nertim** [comment]
I see thank you @weidongxu-microsoft .. @naveedaz does your sign off count as the answer here? Or do we need to something explicit?

**Comment by naveedaz** [comment]
@nertim Can you confirm that the API itself is not new but already used for a while with portal and that whether or not the additional flags property is specified there is no regression in behavior?

**Comment by nertim** [comment]
will try to confirm today.

**Comment by nertim** [comment]
apologies for the delay.. I have confirmed there is no regression in behavior.

**Comment by nertim** [comment]
@weidongxu-microsoft what would be the next steps here?

**Comment by weidongxu-microsoft** [comment]
@nertim 

It would get API review board for approval (adrianhall).

**Comment by nertim** [comment]
@weidongxu-microsoft @adrianhall - Can you please help identify the next steps here?

**Comment by nertim** [comment]
@weidongxu-microsoft @adrianhall - can you please help identify the next steps here?

**Comment by weidongxu-microsoft** [comment]
Here is the template on breaking changes. You can contact JeffreyRichter for approval.

Hi @nertim, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow breaking change policy to request breaking change review and approval before proceeding swagger PR review.
If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ma/swaggerfeedback.

**Comment by nertim** [comment]
@weidongxu-microsoft - The breaking changes are being marked on new optional properties added to the API. If the users do not utilize these new properties, their existing scenarios will continue to work as is. By using the new properties, the user would be able to take advantage of some the newer features we are rolling out.

From a business perspective, the new feature allows users to setup GitHub Action workflows via the existing sourecontrol api and clean up as needed. Again, this is available IF the users decided to include the new added properties in the API call.

**Comment by weidongxu-microsoft** [comment]
> @weidongxu-microsoft - The breaking changes are being marked on new optional properties added to the API. If the users do not utilize these new properties, their existing scenarios will continue to work as is. By using the new properties, the user would be able to take advantage of some the newer features we are rolling out.
> 
> From a business perspective, the new feature allows users to setup GitHub Action workflows via the existing sourecontrol api and clean up as needed. Again, this is available IF the users decided to include the new added properties in the API call.

Thanks for the explanation. But the procedure requires @JeffreyRichter approval before I can do anything.

**Comment by ravbhatnagar** [comment]
please re-add ARM feedback label if you need a review from ARM. removing to reduce noise.

**Pull request closed without merging**